test/eth/TransactionManager.spec.js

```diff
@@ -262,22 +263,11 @@ describe('lifecycle hooks', () => {
     expect(txMgr._tracker._transactions).not.toHaveProperty(drawId);
   });
 
-  test('finalized Tx is set to correct state without without requiring a call to confirm()', async () => {
```

Comment:
@sirromdev do you remember why you removed this test? i think it was actually catching an important regression